docs/basic_usage.md

```diff
@@ -12,7 +12,7 @@ module SignUpFormParams = {
   /* Define the form state */
   type state = {
     password: string,
-    confirmP: string,
+    confirmP: string,
```

Comment:
good catch!, but I think it is `confirmPassword`

Reply:
yes. I think Github is not on my side today :'( . I typed confirmPassword but it kept saving it as confirmP